src/css/utils/variables.styl

```diff
@@ -11,27 +11,10 @@ $mq_vars = {
 // -------------------
 // Typefaces
 // -------------------
-$canela = 'Canela Web'
-$canela-hash = 'canela'
-$publico = 'Publico Text Web'
-$publico-hash = 'publico'
-$atlas = 'Atlas Grotesk Web'
-$atlas-hash = 'atlas'
-
-$sans-fb = Helvetica, Arial, sans-serif
-$sans = $atlas, $sans-fb
-$serif-fb = Georgia, Times, serif
-$serif = $publico, $serif-fb
-$serif-hed-fb = 'Times New Roman', serif
+$sans = Polaris, helvetica, sans-serif;
+$serif = Copernicus, georgia, serif;
 $mono = Menlo, Consolas, Monaco, 'Courier New', monospace
 
-// XXX: quick hack
-$sans = $sans-fb
-$serif = $serif-fb
-$canela = $serif-hed-fb
-$atlas = $sans-fb
-$publico = $serif-fb
```

Comment:
Not a big deal, but without these aliases, I get some errors in the console about trying to load these fonts.

```load-font.js?ea86:15 A {family: "Canela Web", style: "normal", weight: 300, stretch: "normal"}
handleError @ load-font.js?ea86:15
Promise.catch (async)
loadFontGroup @ load-font.js?ea86:41
(anonymous) @ critical.js?e5e6:29
(anonymous) @ critical.js:71
__webpack_require__ @ critical.js:20
(anonymous) @ critical.js:63
(anonymous) @ critical.js:66
load-font.js?ea86:15 A {family: "Publico Text Web", style: "normal", weight: 400, stretch: "normal"}
handleError @ load-font.js?ea86:15
Promise.catch (async)
loadFontGroup @ load-font.js?ea86:41
(anonymous) @ critical.js?e5e6:30
(anonymous) @ critical.js:71
__webpack_require__ @ critical.js:20
(anonymous) @ critical.js:63
(anonymous) @ critical.js:66
load-font.js?ea86:15 A {family: "Atlas Grotesk Web", style: "normal", weight: 400, stretch: "normal"}
```

I'll look into it.

Reply:
Sorry! Didn't even think about that.